apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -89,79 +90,62 @@ export const renderMeterReading = (values: string[], resourceId: string, measure
     return `${getIntegerPartOfReading(values[0])} ${measure}`
 }
 
-const fillMoneyWithSpaces = (substring: string, startIndex: number, spaces: Array<number>) => {
-    const chars = []
-    for (let i = 0; i < substring.length; i++) {
-        const absoluteIndex = startIndex + i
-        const char = substring.charAt(i)
-        if (spaces.includes(absoluteIndex)) {
-            chars.push(MONEY_PARTS_SEPARATOR)
+const getCurrencySymbol = (currencyName) => {
```

Comment:
This function gets the currency mark, like $ for dollar. The reason for this function is because native Intl returns string in this format:

`<currencyMark><value>.<decimalPart> // $ 20.00` but we need to have it this way: `20.00$`

While it is obvious that we can simply take the first symbol and put it in the end of the string, it also might not work with specific locales, which does not have a narrow symbol (ETH, BTC) or for negative numbers

Reply:
Well, yes, but for some annoying reason Intl thinks that russian locale should start with currency mark :(

I think amount is by default more important then currency in our service, since you only work with one currency at the time, so let's leave a template like this: `amount + code` 

![image](https://user-images.githubusercontent.com/33755274/137987369-5190f5ed-502f-4d74-b7da-14fb0d3d8dbf.png)

Reply:
> <currencyMark><value>.<decimalPart> // $ 20.00 but we need to have it this way: 20.00$`
Are we? I guess we should not fix position of currency. Instead we can create locale-depended resolver, which generates value (and use Intl as fallback?)....But not hard-coded

Reply:
Please see the latest commits, the i18n is completly rewritten